SampleApplications/SDK/Opc.Ua.Client/Session.cs

```diff
@@ -849,6 +850,16 @@ public int GoodPublishRequestCount
                 {
                     throw ServiceResultException.Create(StatusCodes.BadConfigurationError, "ApplicationCertificate cannot be found.");
                 }
+
+                // load certificate chain.
+                clientCertificateChain = new X509Certificate2Collection(clientCertificate);
+                List<CertificateIdentifier> issuers = new List<CertificateIdentifier>();
+                configuration.CertificateValidator.GetIssuers(clientCertificate, issuers).Wait();
```

Comment:
>.Wait(); [](start = 89, length = 8)

this is an async function. use await, do not mix wait and and async calls

Reply:
fixed.